### ISSUE TITLE
setup.py file should be compatible with Python 2.7

### ISSUE THREAD

**Issue by astrofrog** [astropy/photutils#793]
In the package template, we've taken the policy that the setup.py file has to remain Python 2-compatible so that users get a nice error message once python_requires is evaluated. Currently, Python 2 users get the obscure error:

```
Traceback (most recent call last):
  File ".\setup.py", line 8, in <module>
    from configparser import ConfigParser
ImportError: No module named configparser
```

**Comment by larrybradley** [comment]
Thanks, @astrofrog.  This is fixed.

**Comment by astrofrog** [comment]
@larrybradley - thanks! Just out of interest, are there plans for a release soon-ish?

**Comment by larrybradley** [comment]
I just released `0.6` before the holidays.  Do you need a quick bugfix release for this issue?

**Comment by astrofrog** [comment]
@larrybradley - no don't worry, but once 0.7 is out I'll be able to build some wheels for photutils (more soon on the infrastructure I'm setting up). Due to the tools I'm using, I can't currently build wheels on Windows due to this issue.
